File: Dante1/Module1.04/Zadanie4.01/main.rs
fn main()
{
    let array:[i32; 10] = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
    
    println!("{:?}", array);
    
    for i in &array
    {
        print!("{} ", i);    
    }
    
    return;
}

File: Dante1/Module1.04/Zadanie4.01/main.zig
const print = @import("std").debug.print;

pub fn main() !void {
	const array =  [10]i32 {0, 1, 2, 3, 4, 5, 6, 7, 8, 9};
	for (array) |i| {
		print("{d} ", .{i});
	}
}
